keps/sig-network/20201010-kube-proxy-v2/README.md

```diff
@@ -0,0 +1,753 @@
+# WORK IN PROGRESS
+<!--
+**Note:** When your KEP is complete, all of these comment blocks should be removed.
+
+To get started with this template:
+
+- [ ] **Pick a hosting SIG.**
+  Make sure that the problem space is something the SIG is interested in taking
+  up. KEPs should not be checked in without a sponsoring SIG.
+- [ ] **Create an issue in kubernetes/enhancements**
```

Comment:
I think you didn't do this part? In particular, the KEP directory name should start with an issue number now, not with a date. (Also, you didn't copy in the `kep.yaml` from the template dir.)

Reply:
ahhh the issue is not the PR, ok got it and willco :)